doc/samples/sample_notify_fifo.sh

```diff
@@ -37,6 +38,7 @@ stopping()
 
 start_shutdown()
 {
+	[[ ! -f "$PID_FILE" || "$(cat "$PID_FILE")" -ne $$ ]] && exit 0
```

Comment:
I don't understand the reason for this. What is the scenario you are protecting against?

Reply:
It was not useful with flock. See next my comment